src/components/EventInfo/Image.jsx

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+
+const Image = styled.img`
+  max-width: 720;
+  max-height: 480;
```

Comment:
Can be removed, defaults to auto

Reply:
Apologies, I will add the appropriate units and remove the width and height from the JSX.